openapi/components/schemas/KycDocument/Settings/KycSettingsAddress.yaml

```
@@ -1,66 +1,75 @@
 type: object
+description: Address proof settings.
 properties:
   weights:
     type: object
-    description: Weights of all matched properties.
+    description: |-
+      Property weights that are used for the KYC document verification process.
+
+      All KYC documents start the verification process with a score of 100.
+      If a check fails, the score is reduced by the corresponding weight.
+      For example, if the `firstName` check weight is set to `5`, and the check fails,
+      the KYC document score becomes `95`.
     properties:
       firstName:
         type: integer
         minimum: 0
         maximum: 100
-        description: Weight of firstName.
+        description: Weight added if the customer's first name is not matched.
       lastName:
         type: integer
         minimum: 0
         maximum: 100
-        description: Weight of lastName.
+        description: Weight added if the customer's last name is not matched.
       line1:
         type: integer
         minimum: 0
         maximum: 100
-        description: Weight of line1.
+        description: Weight added if the customer's address is not matched.
       city:
         type: integer
         minimum: 0
         maximum: 100
-        description: Weight of city.
+        description: Weight added if the customer's city is not matched.
       region:
         type: integer
         minimum: 0
         maximum: 100
-        description: Weight of region.
+        description: Weight added if the customer's region is not matched.
       postalCode:
         type: integer
         minimum: 0
         maximum: 100
-        description: Weight of postalCode.
+        description: Weight added if the customer's postal code is not matched.
       date:
         type: integer
         minimum: 0
         maximum: 100
-        description: Weight of date.
+        description: Weight added if a date is not detected on the document.
       phone:
         type: integer
         minimum: 0
         maximum: 100
-        description: Weight of phone.
+        description: |-
+          Weight added if a phone number is not detected on the document.
       documentSubtype:
         type: integer
         minimum: 0
         maximum: 100
-        description: Weight of documentSubtype.
+        description: |-
+          Weight added if the document is not one of the recognized document subtypes.
+          For a list of available subtypes, see [KycDocumentSubtypes](https://github.com/Rebilly/api-definitions/blob/main/openapi/components/schemas/KycDocument/KycDocumentSubtypes.yaml).
```

```suggestion
        description: Weight added if the document is not one of the recognized document subtypes.
```

I think we don't need the link here.